src/sentry/search/eap/columns.py

```diff
@@ -331,6 +337,33 @@ def simple_measurements_field(field) -> ResolvedColumn:
 }
```

Comment:
Something for either of us to do in a future clean-up, but I think someone should add some sort of validation when constructing `SPAN_COLUMN_DEFINITIONS` so that we validate each internal_name has at least 1 primary_alias